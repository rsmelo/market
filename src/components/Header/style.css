.header {
  display: flex;
  justify-content: center;
  width: 100%;
  padding: 1em 0;
  background-color: var(--color-light-green);
}

.content {
  display: flex;
  width: 100%;
  max-width: 1170px;
  justify-content: space-between;
}

.titleLink:hover,
.titleLink:visited,
.titleLink:active {
  font-size: 28px;
  font-weight: lighter;
  text-decoration: none;
  color: var(--color-white);
}

.cartIcon {
  background-image: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACQAAAAkCAQAAABLCVATAAABFUlEQVRIx2NgGM7gf8N/GFhALYN+/ZejjkH//0+m3IPO/0kHD/+rYTPqFBlGlWMzKJBkY/7818JmEOP/ayQaNAtXOMWRZMyX/xK4DGL5/4AEg+rxxV0O0cY8+8+NzyDO/y+JNCiVUHqqJMqYK/+ZCRnE//8DEQZ5EZPG2wgas4e4zOJAwJh//w2JM2gXAYOaiDPGHaz49/8WYKmACav+WxNnDNP/i2CDplFamEAy7qf/YpQaVAM26DlWb6FCB/wG+RGdRRrwG8T4fz1VDAIb5fu/lmKvUbOWs/9/AViEnvtvQ7osskK5/1/h5Z8MabKoSguRgjOfNFkaGSQLdPR/aOqWJk0WXbHN/7PANsCZ/1akyw5TAAC4GqRAeRYA+wAAAABJRU5ErkJggg==");
  background-repeat: no-repeat;
  width: 36px;
  height: 36px;
  border: none;
  background-color: transparent;
}
