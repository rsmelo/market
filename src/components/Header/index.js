import React from 'react'
import { Link } from 'react-router-dom'

import style from './style.css'

const Header = () => (
  <div className={style.header}>
    <div className={style.content}>
      <h1>
        <Link to="/" className={style.titleLink}>
          Market
        </Link>
      </h1>
      <Link to="/cart">
        <div className={style.cartIcon} />
      </Link>
    </div>
  </div>
)

export default Header
