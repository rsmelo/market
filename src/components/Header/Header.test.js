import React from 'react'
import renderer from 'react-test-renderer'

import Header from './index'

jest.mock('react-router-dom', () => ({ Link: 'Link' }))

describe('Header', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(<Header />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
