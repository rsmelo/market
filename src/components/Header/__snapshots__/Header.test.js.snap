// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<div
  className={undefined}
>
  <div
    className={undefined}
  >
    <h1>
      <Link
        className={undefined}
        to="/"
      >
        Market
      </Link>
    </h1>
    <Link
      to="/cart"
    >
      <div
        className={undefined}
      />
    </Link>
  </div>
</div>
`;
